src/Picqer/Financials/Exact/Connection.php

```diff
@@ -537,6 +559,29 @@ private function parseResponseXml(Response $response)
         }
     }
 
+    /**
+     * @param Response $response
+     *
+     * @throws ApiException
+     *
+     * @return mixed
+     */
+    private function parseDownloadResponseXml(Response $response)
+    {
+        try {
+            if ($response->getStatusCode() === 204) {
+                return [];
+            }
+
+            Psr7\Message::rewindBody($response);
+            $simpleXml = new \SimpleXMLElement($response->getBody()->getContents());
```

Comment:
```suggestion
            $simpleXml = new \SimpleXMLElement($response->getBody()->__toString());
```

If `__toString()` is used, there is no need to manually rewind stream. `StreamInterface::getContents()` behaviour is inconsistent across different psr 7 libraries.

https://www.php-fig.org/psr/psr-7/#13-streams